### PR TITLE
feat: upgrade Kendo UI definitions to 2017.3.913

### PR DESCRIPTION
**Pull request by hadjieva**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by uniqueiniquity** [comment]
Since you are updating to a new major version, please follow the guidelines at https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#i-want-to-update-a-package-to-a-new-major-version

Also, please add tests to reflect your changes.

**Comment by Stamo-Gochev** [comment]
@uniqueiniquity Is it OK for the folder name to contain dots, e.g. `v2017.3.913` or is it preferable to look like `v20173913`?

**Comment by uniqueiniquity** [comment]
@Stamo-Gochev the dots should be fine.

File: types/kendo-ui/index.d.ts [hunk 1]
```diff
@@ -10112,10 +10140,42 @@ declare namespace kendo.dataviz.ui {
...
 
+    interface ChartSeriesItemMarkersFromBorder {
+        color?: string|Function;
+        width?: number|Function;
+    }
+
+    interface ChartSeriesItemMarkersFrom {
+        background?: string|Function;
+        border?: ChartSeriesItemMarkersFromBorder;
+        size?: number|Function;
```
Comment (uniqueiniquity) [inline types/kendo-ui/index.d.ts]: On this and other uses of the type `Function`, please consider using a more specific type as discussed here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]

**Comment by Stamo-Gochev** [comment]
@uniqueiniquity Can you shed some light why the changes result in the [error](https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/278145143#L878):
```
Error: Directory /home/travis/build/DefinitelyTyped/DefinitelyTyped/types/kendo-ui/v2017.3.913 has unused file index.d.ts
```
It looks correct according to the example: https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/history/v2

As for the case with `Function` - our TS definitions are dependent on our docs:
http://docs.telerik.com/kendo-ui/api/javascript/ui/grid#configuration-columns.template

Currently there is no information about the return type of the function:
```
columns.template String | Function
```
which prevents us from upgrading to the required format right now, but we have ideas for improving the TS definitions process as soon as we can.

**Comment by uniqueiniquity** [comment]
@andy-ms Do you know why this error might be happening?

**Comment by andy-ms** [comment]
We only support major versions that are natural numbers, just like NPM does.
In this case it doesn't seem that this is actually a major version change -- 2017.2.621 to 2017.3.913 . So it shouldn't be necessary to create a new directory.

**Comment by uniqueiniquity** [comment]
Thanks, @andy-ms. My mistake, I apologize for the confusion.

**Comment by Stamo-Gochev** [comment]
@uniqueiniquity @andy-ms Thanks for the clarifications. The TS definitions are now updated.

**Review by uniqueiniquity — APPROVED** [review]

**Pull request merged**